examples/tts/fastspeech_durations.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2019 NVIDIA Corporation
```

Comment:
same, here and (perhaps in other new files which you've added) - use the standard licence/copyright header